sites/fast-website/src/public/index.html

```diff
@@ -98,19 +99,19 @@ <h2 class="section-heading">Join the community</h2>
           </div>
         </section>
 
-        <footer class="site-footer">
-          <site-navigation>
-            <ul>
-              <li><fast-anchor appearance="lightweight" href="#">License</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="#">Privacy & Cookies</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="#">Terms of Use</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="#">&copy; 2020 Microsoft</fast-anchor></li>
-            </ul>
-          </site-navigation>
-          <fast-anchor class="logo" appearance="lightweight" href="#">
-            <%= require("svg/icon-brand.svg") %>
-          </fast-anchor>
-        </footer>
+          <div class="footer">
+            <site-navigation>
+              <ul>
+                <li><fast-anchor appearance="lightweight" href="https://github.com/microsoft/fast-dna/blob/master/LICENSE">License</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="https://privacy.microsoft.com/en-US/privacystatement">Privacy & Cookies</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx">Terms of Use</fast-anchor></li>
+                <li><p>&copy; 2020 Microsoft</p></li>
+              </ul>
+            </site-navigation>
+            <fast-anchor class="logo" appearance="lightweight" href="#">
```

Comment:
hash link

Reply:
What url should I set for the logo link?

Reply:
Probably either `/` or `https://fast.design`

Reply:
thanks, using `/`